cibuildwheel/linux.py

```diff
@@ -95,6 +95,7 @@ def build(project_dir, package_name, output_dir, test_command, test_requires, be
 
                 # we're all done here; move it to output
                 mv "$delocated_wheel" /output
+                chown {uid}:{gid} /output/$(basename $delocated_wheel)
```

Comment:
Since all variable expansions before are also quoted, can we still change this to `"/output/$(basename "$delocated_wheel")"` ?